### DISCUSSION TITLE
Can't run shell

### DISCUSSION THREAD

**Discussion by qrilka** [helix-editor/helix#6812] [Q&A] [answered]
On https://docs.helix-editor.com/commands.html I see

> `:run-shell-command`, `:sh`	Run a shell command

But when I try to enter `:sh` I get

> no such command: `sh`

And I can't find any page explaining what could be the reason and how to run a shell command.

```
$ hx --version
helix 22.03 (d4e45fd4)
```

**Comment by the-mikedavis** [comment, marked as answer]
You're running an old build: 22.03 is March of 2022. `:run-shell-command`/`:sh` first showed up in 22.05 https://github.com/helix-editor/helix/pull/1682

**Reply by qrilka** [reply]
Thanks Mike,
I've tried searching through the CHANGELOG and saw the corresponding entry
> `:run-shell-command` which does not interact with selections

was just too confusing: it looked like some improvement and not a new command
